lib/test_runner.zig

```diff
@@ -1,3 +1,4 @@
+//! Default test runner for unit tests.
```

Comment:
This was added to prevent people misunderstanding [what the test_runner ist for like in this PR](https://github.com/ziglang/zig/pull/16522#issuecomment-1648177116).

Reply:
@andrewrk done. I was so free to add this one line to specify the purpose of the default test runner.